tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -850,3 +851,40 @@ def test_read_staged_file_no_commit(session):
     assert Utils.is_active_transaction(session)
     session.sql("commit").collect()
     assert not Utils.is_active_transaction(session)
+
+
+@pytest.mark.skipif(
```

Comment:
why not move this to simplifier suite?

Reply:
Loading from files has some fixtures defined in this module. It's not as simple as other tests so I put these tests together with other reader tests.